src/failures/src/a320.ts

```diff
@@ -15,6 +15,14 @@ export const A320Failure = Object.freeze({
     YellowReservoirReturnLeak: 29008,
     LeftPfdDisplay: 31000,
     RightPfdDisplay: 31001,
+    LgciuPowerSupply1: 32000,
+    LgciuPowerSupply2: 32001,
+    LgciuInternalError1: 32002,
+    LgciuInternalError2: 32003,
+    GearProxSensorDamageGearUplockLeft1: 32004,
+    GearProxSensorDamageDoorDownlockRight2: 32005,
+    GearProxSensorDamageGearUplockCenter1: 32006,
+    GearProxSensorDamageDoorUplockLeft2: 32007,
```

Comment:
It seems like there are some combinations missing here. Will you be adding those in a separate PR?

Reply:
There's already too much of them there and efb 2 display is getting crazy.... So yeah will add tons of them when efb GUI can handle this